### ISSUE TITLE
Backing up system

### ISSUE THREAD

**Issue by Jonbj** [Alfresco/alfresco-docker-installer#63]
What's the best way to backup system?
What about to improve dockerfile with backup?

**Comment by aborroy** [comment]
I guess backup should be an external process. Not sure this must be a feature provided by this project.

I've created a sample process to create and restore backups in https://hub.alfresco.com/t5/alfresco-content-services-blog/creating-and-restoring-backups-for-alfresco-with-docker-compose/ba-p/305070#M2516

Is that enough?

**Comment by Jonbj** [comment]
Great post! Thank you!

**Comment by aborroy** [comment]
Thanks, @DejmalDavid 

Do you want to contribute your tool to Alfresco Docker Installer project? We could provide it to the Community by default.

**Comment by DejmalDavid** [comment]
Yes, it would be nice if the community would use me tool. I was searching about some automatic backup for dockerized Alfresco but didn't find any so i start write my own. But there's still more TODOs than done and it need revision from someone else than me.
